source/index.html.erb

```diff
@@ -596,37 +596,91 @@ per_page: 10
 
     <section class="resume-section p-3 p-lg-5 d-flex flex-column section-scroll" id="followus" data-nav-color="yellow">
       <div class="my-auto">
-        <div class="row justify-content-center">
-          <h2 class="mb-5" id="follow">
+        <div>
+          <h2 class="mb-5" id="follow"  align="center">
             <%= data.i18n.en.sections.contact.title %>
           </h2>
+          <p class="mb-5">
+            We want to make it super-easy for Peatio users and contributors to talk to us and connect with each other, to share ideas, solve problems and help make Peatio awesome.
+          </p>
+          <h4 class="mb-5"  align="center">
+            Communications
+          </h4>
+          <p class="mb-5">
+            Here are the communication channels we're running currently - we'd love to hear from you on any one of them!
+          </p>
         </div>
-        <div class="row">
-          <div class="col-sm-12 col-md-4">
-            <div class="follow-info">
-              <p>We want to make it super-easy for Peatio users and contributors to talk to us and connect with each other, to share ideas, solve problems and help make Peatio awesome. Here are the main channels we're running currently, we'd love to hear from you on one of them</p>
-            </div>
+
+        <div class="row mb-2">
+          <div class="col-sm-12 col-md-2 text-center  mb-1">
+            <i class="fab fa-github fa-6x github"></i>
+          </div>
+          <div class="col-sm-12 col-md-7 centeredContent">
+             <p>
+              Ready to dive into the code and get your feet wet? Welcome to our GitHub!
+             </p>
+          </div>
+          <div class="col-sm-12 col-md-3 centeredContent mb-3">
+              <a href="https://github.com/rubykube" target="_blank" class="btn btn-dark btn-block btnLowerFont">GitHub branch</a>
+          </div>
+        </div>
+
+        <div class="row mb-2">
+          <div class="col-sm-12 col-md-2 text-center mb-1">
+            <i class="fab fa-telegram fa-6x telegram"></i>
           </div>          
-          <div class="col-sm-12 col-md-1">
-            <div class="line"></div>
-          </div>
-          <div class="col-sm-12 col-md-5">         
-            <div class="follow-block">
-              <p> You can find guides, recipes, questions, and answers from Snowplow users including the Rubykube team.</p>
-              <div id="follow-btn">
-                <a href="http://discuss.rubykube.io" target="_blank" class="telegramim_button telegramim_shadow telegramim_pulse" style="font-size:12px;width:140px;background:#EE4D3E;box-shadow:1px 1px 5px #F48F86;color:#212529;border-radius:34px;" title="discuss">discuss.rubykube.io</a>
-              </div>
-            </div>
+          <div class="col-sm-12 col-md-7 centeredContent">
```

Comment:
Bootstrap already has classes for centering content, check the doc please (see about flex properties)